sos/report/plugins/__init__.py

```diff
@@ -2421,6 +2421,10 @@ def add_journal(self, units=None, boot=None, since=None, until=None,
         if output:
             journal_cmd += output_opt % output
 
+        if otherargs:
+            for arg in otherargs:
+                journal_cmd += ' {}'.format(arg)
+
```

Comment:
As mentioned above, this is fairly error prone based on what gets handed to `add_journal()`. The option string formatting should be performed here, not in individual plugins.